dbms/src/Storages/DeltaMerge/tests/gtest_dm_delta_merge_store.cpp

```diff
@@ -178,7 +178,7 @@ class DeltaMergeStoreRWTest
         return s;
     }
 
-    std::pair<RowKeyRange, std::vector<PageId>> genDMFile(DMContext & context, const Block & block)
+    std::pair<RowKeyRange, PageIds> genDMFile(DMContext & context, const Block & block)
```

Comment:
I prefer to add it in the follow-up PR. Cause writing ut with `GlobalPathPool` is not easy.

otherwise, already have test cover in V3/gtest_page_storage_mix_mode.cpp.